doc/introduction/compiling_circuits.rst

```diff
@@ -373,6 +346,192 @@ From here, we can iterate through the stages of decomposition:
 ───H†─│────────────╰●───────────────┤  
 ──────╰●────────────────────────────┤  
 
+Custom Operator Decomposition
+-----------------------------
+
+When executing QNodes on a device, PennyLane will automatically decompose
+gates that are unsupported by the device using built-in decomposition rules.
+
+In addition, you can provide custom decomposition rules to be used.
+
+The default behaviour in PennyLane
+versus that of the new decompositions system (enabled with ``qml.decomposition.enable_graph``)
+differ in the following ways when it comes to injecting custom decompositions for
+operators:
+
++-------------------+------------------------------+-----------------------------+--------------------------------------+
+|                   | QNode-level decomposition    | Device-level decomposition  | Multiple decompositions per operator |
++===================+==============================+=============================+======================================+
+| ``disable_graph`` | No                           | Yes                         | No                                   |
++-------------------+------------------------------+-----------------------------+--------------------------------------+
+| ``enable_graph``  | Yes                          | Yes                         | Yes                                  |
++-------------------+------------------------------+-----------------------------+--------------------------------------+
+
+Behaviour with qml.decomposition.disable_graph
```

Comment:
Since the new system is disabled by default, this is just the default behaviour. Maybe instead of phrasing it like enable_graph vs. disable_graph, phrase it like current behaviour / experimental behaviour with graph enabled?